LVT/LVT.Services/InitiativePresenter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using LVT.LVT.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace LVT.LVT.Services
+{
+    public class InitiativePresenter : IVisualizer<Initiative>
+    {
+        //private IVisualizer<Epic> _epicpre;
+        //public InitiativePresenter(IVisualizer<Epic> epicpre)
+        //{
+        //    _epicpre = epicpre;
+        //}
+
+        public string VisualizeToString(Initiative initiative, string parentNode)
+        {
+            string result = "[{ v: '" + initiative.NodeID + "', f: 'Initiative" + "<div style=\"font-style:italic\">" + initiative.Title + "</div>'}, " + $"'{parentNode}']";
+
+            if (initiative.Measures != null && initiative.Measures.Count() >= 1)
+            {
+                Console.WriteLine(result);
+                result = result + " , " + ProcessMeasures(initiative, initiative.NodeID);
+                Console.WriteLine(result);
```

Comment:
Do we need the `Console.WriteLine`s?

Reply:
While we are still working, I'm leaving this in as a means to visually confirm that the program is doing its thing.